src/main/java/com/kapas/workorder/controller/WorkflowController.java

```diff
@@ -0,0 +1,36 @@
+package com.kapas.workorder.controller;
+
+import com.kapas.KapasApplication;
+import com.kapas.user.model.LoggedInUser;
+import com.kapas.user.model.Login;
+import com.kapas.user.service.UserService;
+import com.kapas.workorder.model.Workflow;
+import com.kapas.workorder.service.WorkflowService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+public class WorkflowController {
+
+    private static final Logger logger = LoggerFactory.getLogger(WorkflowController.class);
+
+    private final WorkflowService workflowService;
+
+    public WorkflowController(WorkflowService workflowService) {
+        this.workflowService = workflowService;
+    }
+
+    @PostMapping(value = "/getWorkflowByWorkflowId")
```

Comment:
since we are fetching values it should be @getmapping

Also rest endpoint should not contain 'get'

Reply:
done